components/forecast/AvalancheTab.tsx

```diff
@@ -102,14 +102,14 @@ export const AvalancheTab: React.FunctionComponent<AvalancheTabProps> = React.me
       <Card borderRadius={0} borderColor="white" header={<Heading>Avalanche Danger</Heading>}>
         <AvalancheDangerTable date={parseISO(forecast.published_time)} current={currentDanger} outlook={outlookDanger} elevation_band_names={elevationBandNames} />
       </Card>
-      <Card borderRadius={0} borderColor="white" header={<Heading>Avalanche Problems</Heading>}>
+      <CollapsibleCard startsCollapsed borderRadius={0} borderColor="white" header={<Heading>Avalanche Problems</Heading>}>
```

Comment:
nit: do we want to collapse all of the problems at once, or each individual problem?

Reply:
🤷 they're separate in [Figma](https://www.figma.com/file/Bai2w7x0vUakL5WRjPGRg7/App---V3?node-id=97%3A89887&t=VvefbQ7LI0dqsU2z-0), I assume this is one of those things that might change with feedback.

Reply:
Should we have it per-problem for now (I imagine designers will call it out on review?)

Reply:
Tracked in https://github.com/stevekuznetsov/avalanche-forecast/issues/45